### DISCUSSION TITLE
Always new file name with date/time

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#120] [Q&A] [answered]
Even if a named tour is loaded, on exporting as kurviger file always a new file name is written. I find this not that good. Before when exporting the last used tour name was re-used for exporting. Can this be made again like before?

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom thanks for the suggestion!

Note that route names are essentially their file names.
New Android does not like files, making our lives difficult to read file names / extensions.

Nevertheless, I will see what is possible! :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
Thanks

**Reply by 0709wiwiwi** [reply]
@devemux86 .  
Usefully to be used or maybe not ? (kurviger ?)
In gpx the (nav)track name is embedded into gpx file itself.
The file name and the track name are so independent.

**Reply by devemux86** [reply]
@0709wiwiwi 

Some gpx files may contain a name (or file name) inside a tag, but this is not certain or expected.

**Reply by 0709wiwiwi** [reply]
Cruiser as the gpx file creator (source) anyway can (**must not**) freely add extra useful info in a gpx (nav)track ;-)
